lib/ditto.ts

```diff
@@ -116,10 +151,14 @@ const executeCommand = async (
       return removeProject();
     }
     case "generate-suggestions": {
-      return generateSuggestions();
+      return generateSuggestions({
+        ...(options.directory ? { directory: options.directory } : {}),
+      });
     }
     case "replace": {
-      return replace(options);
+      return replace(options.args, {
+        ...(options.lineNumbers ? { lineNumbers: options.lineNumbers } : {}),
```

Comment:
If `options` can actually be `any` (e.g. `null` or `undefined`), then we'd need to chance this to `options?.lineNumbers`